src/dotnet/APIView/APIView/Model/V2/ReviewToken.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace APIView.Model.V2
+{
+    /// <summary>
+    /// Token corresponds to each component within a code line. A separate token is required for keyword, punctuation, type name, text etc.
+    /// </summary>
+    public class ReviewToken
+    {
+        public ReviewToken() { }
+        public ReviewToken(string value, TokenKind kind)
+        {
+            Value = value;
+            Kind = kind;
+        }
+        public TokenKind Kind { get; set; }
+        public string Value { get; set; }
+        /// <summary>
+        /// NavigationDisplayName property is to add a short name for the token that will be displayed in the navigation object.
+        /// </summary>
+        public string NavigationDisplayName { get; set; }
+        /// <summary>
+        /// navigateToId should be set if the underlying token is required to be displayed as HREF to another type within the review.
+        /// </summary>
+        public string NavigateToId { get; set; }
+        /// <summary>
+        /// set skipDiff to true if underlying token needs to be ignored from diff calculation. 
+        /// For e.g. package metadata or dependency versions are usually not required to be excluded when comparing two revisions to avoid reporting them as API changes
+        /// </summary>
+        public bool? SkipDiff { get; set; }
+        /// <summary>
+        /// This is set if API is marked as deprecated
+        /// </summary>
+        public bool? IsDeprecated { get; set; }
+        /// <summary>
+        /// Set this to false if there is no suffix space required before next token. For e.g, punctuation right after method name
+        /// </summary>
+        public bool? HasSuffixSpace { get; set; }
+        /// <summary>
+        /// Set isDocumentation to true if current token is part of documentation
+        /// </summary>
+        public bool? IsDocumentation { get; set; }
+        /// <summary>
+        /// Language specific style css class names
+        /// </summary>
+        public HashSet<string> RenderClasses { get; set; } = new HashSet<string>();
```

Comment:
Does this typically contain a small number of strings? If so, it may be more efficient as a simple array. I don't see anywhere in this PR where it is used as a HashSet.